.github/workflows/shfmt.yml

```diff
@@ -0,0 +1,25 @@
+name: Shfmt
+
+# shfmt formats shell programs
+# https://github.com/mvdan/sh#shfmt
+
+permissions:
+  contents: read
+
+on: [workflow_dispatch, pull_request, push]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    name: Shfmt - Lint
+    runs-on: ubuntu-20.04
+    container:
+      image: ghcr.io/build-trust/ockam-builder@sha256:e43dd94652096b03cc472a3c709c7335e8b166cab77b7a7b56f88fa38f3d24cc
```

Comment:
Let's use the latest image SHA `sha256:9598b7bc30bb23e9e64cd95cb96a30af9695e6a16ce1834e6c17c3c62bc81c2e` the current doesn't have `shfmt` installed.

Reply:
Fixed.